packages/core/src/wallet/IndyWallet.ts

```diff
@@ -56,6 +56,10 @@ export class IndyWallet implements Wallet {
   }
 
   public get masterSecretId() {
+    if (this.walletConfig?.masterSecretId) {
```

Comment:
Considering the checks that are done in line 63, for consistency with other conditions, maybe we could avoid this if sentence and simply return `this.walletConfig.masterSecretId ?? this.walletConfig.id`